is_triangle.py

```diff
@@ -0,0 +1,11 @@
+"""Is_Triangle."""
+
+
+def is_triangle(a, b, c):
+    """Test if given numbers can be valid sides of a triangle."""
+    if min([a, b, c]) <= 0:
+        return False
+    elif a + b <= c or a + c <= b or b + c <= a:
+        return False
+    else:
```

Comment:
A trailing `else` operator is unnecessary here, since every other possibility will exit the function.